src/main/java/Parser.java

```diff
@@ -0,0 +1,37 @@
+public class Parser {
+    private String[] parsedInput;
+
+    public Parser(String line) throws InvalidCommandException, MissingDescriptionException {
```

Comment:
It would help to add [javadoc comments](https://se-education.org/guides/conventions/java/basic.html#comments) here to explain the checks done by the constructor.